code/addons/test/src/vitest-plugin/index.ts

```diff
@@ -109,14 +109,15 @@ export const storybookTest = async (options?: UserOptions): Promise<Plugin> => {
     getStoryGlobsAndFiles(presets, directories),
     presets.apply('framework', undefined),
     presets.apply('env', {}),
-    presets.apply('viteFinal', {}),
+    presets.apply<{ plugins?: Plugin[] }>('viteFinal', {}),
```

Comment:
From a discussion with @yannbf:
Because this change brings up plugins from presets (frameworks, addons, etc) we need to do the following:
- Remove setting up framework plugins from sb add postinstall
- Remove reference of such plugins from the docs
- Add a migration note regarding this change for people who already have a set up that includes these plugins